lib/dkdeploy/constants.rb

```diff
@@ -66,23 +66,7 @@ def datetime_format
     #
     # @return [String]
     def default_ignore_tables
-      %w[
-        cache_extensions cache_hash cache_imagesizes cache_md5params
-        cache_pages cache_pagesection cache_sys_dmail_stat cache_treelist cache_typo3temp_log
-        cachingframework_cache_hash cachingframework_cache_hash_tags cachingframework_cache_pages
-        cachingframework_cache_pages_tags cachingframework_cache_pagesection
-        cachingframework_cache_pagesection_tags sys_workspace_cache sys_workspace_cache_tags
-        tt_news_cache tt_news_cache_tags tx_extbase_cache_object tx_extbase_cache_object_tags
-        tx_extbase_cache_reflection tx_extbase_cache_reflection_tags tx_realurl_chashcache
-        tx_realurl_pathcache tx_realurl_urldecodecache tx_realurl_urlencodecache be_users be_sessions
-        sys_domain fe_users fe_sessions fe_session_data
-        cf_cache_hash cf_cache_hash_tags cf_cache_pages cf_cache_pages_tags cf_cache_pagesection
-        cf_cache_pagesection_tags cf_cache_rootline cf_cache_rootline_tags cf_extbase_datamapfactory_datamap
-        cf_extbase_datamapfactory_datamap_tags cf_extbase_object cf_extbase_object_tags
-        cf_extbase_reflection cf_extbase_reflection_tags cf_extbase_typo3dbbackend_queries
-        cf_extbase_typo3dbbackend_queries_tags cf_extbase_typo3dbbackend_tablecolumns
-        cf_extbase_typo3dbbackend_tablecolumns_tags
-      ]
+      %w[]
     end
 
     # List of table names to be ignored when dumping from database defined via Capistrano variable or environment variable
```

Comment:
Please change configuration variable `additional_ignore_tables` from type string to array